### DISCUSSION TITLE
How to configure the gateway address of Data Network for N6 interface

### DISCUSSION THREAD

**Discussion by s5uishida** [edgecomllc/eupf#449] [Q&A] [answered]
Hi,

How do I configure the gateway address of data network for the N6 interface?

Thank you in advance.

**Comment by pirog-spb** [comment, marked as answer]
Hi @s5uishida ,

eUPF uses linux routing table during packet forwarding. Configure N6 gateway by means of linux `ip route ...` commands.

**Reply by s5uishida** [reply]
Hi @pirog-spb 

This inquiry has also been resolved.

Thank you very much!
